db/schema.rb

```diff
@@ -13,6 +13,7 @@
 ActiveRecord::Schema.define(version: 2020_11_30_093110) do
 
   # These are extensions that must be enabled in order to support this database
+  enable_extension "pgcrypto"
```

Comment:
What does this do?

Reply:
Required for UUID generation

Reply:
Ooh, what? That's weird - I thought we added uuids (and tests that should make use of objects using uuids as ids) a while ago :/

Or is this enabling uuids at a database level when our old code does it on application level?

Basically, I'm wondering how did you get an error that needed this fix?

Reply:
Could be dependent on the db version

Reply:
This enables the feature in the db that your code depended on

Reply:
It's off by default on my local db and the db provided by the github action